core/templates/dev/head/pages/library/library.html

```diff
@@ -204,6 +204,28 @@ <h2 ng-class="{'active': activeGroupIndex === $index}" class="oppia-library-grou
     </div>
   </div>
 
+  <script type="text/ng-template" id="modals/setDashboard">
+    <div class="modal-body oppia-set-dashboard-modal">
+      <div ng-if="!defaultDashboardSelected">
+        <p style="padding-top: 30px; padding-bottom: 20px; text-align: center;">
+          Hi! We recently launched the learner dashboard. Keeping track of what you've learned will be easier than ever!
+        </p>
+        <div style="text-align: center;">
+          <button type="button" class="btn oppia-set-dashboard-modal-button oppia-transition-200" ng-click="setDashboard('creator')">Set creator dashboard as default</button>
+          <button type="button" class="btn oppia-set-dashboard-modal-button oppia-transition-200" ng-click="setDashboard('learner')">Set learner dashboard as default</button>
+        </div>
+      </div>
+      <div ng-if="defaultDashboardSelected">
+        <p style="padding-top: 30px; padding-bottom: 20px; text-align: center;">
+          Your default dashboard is now <[defaultDashboardName]>. You can change this anytime in the 'Preferences' section.
```

Comment:
How about this? `When you log in, you'll now see the <[defaultDashboardName]> by default. You can change this anytime on the 'Preferences' page.`